python/README.md

```diff
@@ -40,8 +40,8 @@ Quantum with all its dependencies:
 ```console
 conda create -y -n cuda-quantum python=3.10 pip
 conda install -y -n cuda-quantum -c "nvidia/label/cuda-11.8.0" cuda
-conda install -y -n cuda-quantum -c conda-forge mpi4py openmpi cxx-compiler
-conda env config vars set -n cuda-quantum LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$CONDA_PREFIX/envs/cuda-quantum/lib"
+conda install -y -n cuda-quantum -c conda-forge mpi4py openmpi cxx-compiler cuquantum~=23.10
+conda env config vars set -n cuda-quantum LD_LIBRARY_PATH="$CONDA_PREFIX/envs/cuda-quantum/lib:$LD_LIBRARY_PATH"
```

Comment:
Note that regardless of the uncertainty about the above change, this change is important because if the user has multiple versions of CUDA installed on the host, this new change will ensure that the CUDA we just installed above (cuda-11.8.0) will take precedence.